api/modules/publishedProjects/model.js

```diff
@@ -0,0 +1,31 @@
+var BaseModel = require('../../classes/base_model');
+
+var Projects = require('../projects/model');
+
+var instanceProps = {
+  tableName: 'publishedProjects',
+  project: function() {
+    return this.belongsTo(Projects);
```

Comment:
Doesn't this create an FK in the publishedProjects table for Projects? Shouldn't it be the other way around?